lib/tests/post_iris_call_receive_headers.js

```diff
@@ -0,0 +1,53 @@
+const expect = require('chai').expect;
+const { getRskTransactionHelper } = require('../rsk-tx-helper-provider');
+const { getBtcClient } = require('../btc-client-provider');
+const { getBridge, getLatestActiveForkName } = require('../precompiled-abi-forks-util');
+const { activateFork, sendTxWithCheck, getLatestForkName } = require('../rsk-utils');
+const { ensure0x } = require('../utils');
+
+let rskTxHelper;
+let btcTxHelper;
+
+/**
+ * Takes the blockchain to the required state for this test file to run in isolation.
+ */
+const fulfillRequirementsToRunAsSingleTestFile = async () => {
+  const forkName = process.env.FORK_NAME || getLatestForkName().name;
+  await activateFork(Runners.common.forks[forkName]);
+};
+
+const execute = (description, getRskHost) => {
+  describe(description, function() {
+
+      before(async () => {
+        rskTxHelper = getRskTransactionHelper(getRskHost());
+        btcTxHelper = getBtcClient();
+
+        if(process.env.RUNNING_SINGLE_TEST_FILE) {
+          await fulfillRequirementsToRunAsSingleTestFile();
+        }
+      });
+    
+      it('Calling receiveHeaders method with regular user should not increment BTC blockchain size', async () => {
+        const bridge = getBridge(rskTxHelper.getClient(), await getLatestActiveForkName());
+        const blockNumberInitial = await bridge.methods.getBtcBlockchainBestChainHeight().call();
+        const cowAddress = await rskTxHelper.newAccountWithSeed('cow');
+        const blockHashes = await btcTxHelper.mine();
```

Comment:
Should we also all call updateBridge here as in `receiveHeader` test to make sure that the Bridge is up to date and that we a re informing a valid block?

Reply:
not sure, it was in the original test for receiveHeader but not int the receiveHeaders one... @jeremy-then ??